projects/ngx-cookiebot/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@halloverden/ngx-cookiebot",
+  "name": "@cubesoft/ngx-cookiebot",
```

Comment:
We'd like the package to still live under our namespace, please.

Reply:
Yes sure, I accidentally commited that one, sorry